src/services/helpers.ts

```diff
@@ -113,3 +119,9 @@ export const getProgress = async (profession: Discipline | null): Promise<number
   const totalExercises = profession.leafDisciplines.length * EXERCISES.length
   return doneExercises / totalExercises
 }
+
+export const loadTrainingsSet = async (disciplineId: number): Promise<ServerResponseDiscipline> => {
```

Comment:
@steffenkleinle do you really think this is a good solution to put it here? We do not have load-from-server-stuff in this file, actually. But if you prefer so, think this is a good solution, fine for me, though.

Reply:
Not sure if this is the right location, I just thought it may make sense to move it to a separate function and had our `useLoadDocuments` files and so on in mind.